redfish-core/lib/account_service.hpp

```diff
@@ -1935,6 +1935,16 @@ inline void processAfterGetAllGroups(
             }
             continue;
         }
+
+        // Remove the ipmi group.  Also Remove "ssh" if the new
+        // user is not an Administrator.
+        if ((grp == "ipmi") || ((grp == "ssh") && (roleId != "priv-admin")))
```

Comment:
NOTE: I changed the roleId to actually work.. 
I assume this is what we actually want? @joseph-reynolds @baemyung